aggsender/flows/flow_base.go

```diff
@@ -404,14 +410,22 @@ func (f *baseFlow) getImportedBridgeExits(
 	return importedBridgeExits, nil
 }
 
+// getStartLER returns the last local exit root (LER) based on the configuration
+func (f *baseFlow) getStartLER() common.Hash {
+	if f.cfg.StartLER == (common.Hash{}) {
+		return zeroLER
```

Comment:
I was about to propose simplification, but I was thinking that the `zeroLER` is `0x0`, since it is not the case, wdyt about renaming it to `emptyLER`?